tests/e2e/specs/aotf.js

```diff
@@ -56,10 +144,28 @@ function mockWorkflowService () {
 }
 
 describe('Api On The Fly', () => {
+  beforeEach(() => {
+    cy
+      .intercept('/graphql', {
+        data: {
+          __schema: {
+            mutationType: {
+              fields: MUTATIONS
+            },
+            types: TYPES
+          }
+        }
+      })
+      .as('IntrospectQuery')
+  })
```

Comment:
Cypress allows us to intercept any HTTP requests in the e2e tests. This way, we can avoid mocking the `WorkflowService`, and instead just pretend the server returned our mutations instead of the default ones.